kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -662,6 +660,17 @@ define([
                     return addRow(paramRow);
                 })
             )
+                .then(() => {
```

Comment:
Normally, when field widgets fire up, they validate themselves with their initial value. We toggle a boolean here that stops that, and use the Spec object to validate every field at once. More details in `doBulkValidation` below.